extensions/notebook/package.nls.json

```diff
@@ -33,8 +33,8 @@
 	"title.saveJupyterBook": "Save Book",
 	"title.trustBook": "Trust Book",
 	"title.searchJupyterBook": "Search Book",
-	"title.SavedBooks": "Saved Books",
-	"title.UnsavedBooks": "Unsaved Books",
+	"title.SavedBooks": "Notebooks",
+	"title.UnsavedBooks": "Provided Books",
```

Comment:
Did this name come from PMs? Is there a way we could make the name a little more generic? I'm just worried that at some point we want to add books to that viewlet that aren't "provided" (maybe users can open their own untitled books or something). 

(I also just think "Provided Books" sounds weird and people aren't going to necessarily understand what that means and how to populate the section with anything)

And actually to continue my train of thought - maybe it would be better if we had a menu action here that when clicked would show you a list of the books that are "provided". Because right now it's very unclear how one finds existing books to open...

Reply:
@yualan suggested the names. We probably will expose actions for user discoverability, for this release it's out of scope. But we're definitely looking into how to better the experience.